src/components/Input/__tests__/Input.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+
+import Input from '../Input.js';
+
+describe('COMPONENT - Input', () => {
+  describe('Input without error', () => {
+    let component;
+
+    beforeEach(() => {
```

Comment:
I would skip this **beforeEach** and **afterEach** declarations and just for clarity declare props in it block:
```
it('renders correctly', () => {
  const props = {
    ...
  };
  
  expect...
});
```

Reply:
Done